app/src/test/java/com/github/versus/ListUserConverterTest.java

```diff
@@ -61,94 +61,115 @@ private static List<VersusUser> buildTestUsers() {
         builder.setCity("FootballClub");
         builder.setZipCode(99);
         builder.setPreferredSports(preferredSports3);
-        VersusUser RegularGuy = builder.build();
-        return Arrays.asList(JamesBond, MohammedAli, RegularGuy);
+        VersusUser RegularGuy= builder.build();
+        return Arrays.asList(JamesBond,MohammedAli,RegularGuy);
     }
-
-    private static String buildTestString() {
-        return "007|James|Bond|NotInfiltrated|notfakemail@gmail.com|0000077777|5|London|700|MARTIALARTS, CLIMBING, CRICKET, JUDO, GOLF, SURFING, WRESTLING[u]1|Mohammad|Ali|TheGoat|IamTheDancingMaster@gmail.com|0000000001|5|Ring|1|BOXING[u]999|Regular|Guy|BasicGuy|ILoveBallSports@gmail.com|999999999|3|FootballClub|99|FOOTBALL, BASKETBALL";
```

Comment:
not sure if it's not a fake mail ;-)

Reply:
I was hoping someone will notice this joke 😂